lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java

```diff
@@ -116,18 +144,22 @@ Classification updateClassification(Classification classification)
   /**
    * This method provides a query builder for querying the database.
    *
-   * @return a {@link ClassificationQuery}
+   * @return a {@linkplain ClassificationQuery}
    */
   ClassificationQuery createClassificationQuery();
 
   /**
-   * Creating a new {@link Classification} with unchangeable default values. It will be only
-   * generated and is not inserted until CREATE-call.
+   * "Creating a new {@link Classification} initialized with unchangeable values for {@linkplain
+   * Classification#getKey() key}, {@linkplain Classification#getDomain() domain} and {@linkplain
+   * Classification#getType() type}. The classification will not be inserted into the database until
+   * {@linkplain ClassificationService#createClassification(Classification)} call."
    *
```

Comment:
why did you quote this?

Reply:
It wasn't intended :) I forgot to delete "" after copying